qiskit/algorithms/optimizers/l_bfgs_b.py

```diff
@@ -49,10 +48,9 @@ class L_BFGS_B(SciPyOptimizer):  # pylint: disable=invalid-name
     # pylint: disable=unused-argument
     def __init__(
         self,
-        maxfun: int = 1000,
+        maxfun: int = 15000,
         maxiter: int = 15000,
         ftol: float = 10 * np.finfo(float).eps,
-        factr: Optional[float] = None,
```

Comment:
Seems this delete should have been done as part of #8210 but at least you caught it here!